master/internal/api_runs_intg_test.go

```diff
@@ -0,0 +1,282 @@
+//go:build integration
+// +build integration
+
+package internal
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+
+	"github.com/determined-ai/determined/master/internal/db"
+	"github.com/determined-ai/determined/master/pkg/model"
+	"github.com/determined-ai/determined/master/pkg/ptrs"
+	"github.com/determined-ai/determined/master/pkg/schemas"
+	"github.com/determined-ai/determined/master/pkg/schemas/expconf"
+	"github.com/determined-ai/determined/proto/pkg/apiv1"
+)
+
+// nolint: exhaustruct
+func createTestExpForRun(
```

Comment:
do we need this helper or can we use something like

```createTestExpWithProjectID```

Reply:
Whoops this was left over from when I was modifying the hparams in the experiment, before doing so in the task. Don't need this function